cmgd/cmgd_bcknd_adapter.c

```diff
@@ -801,16 +801,45 @@ static int cmgd_bcknd_adapter_resume_writes(struct thread *thread)
 	return 0;
 }
 
+static void cmgd_bcknd_iter_and_get_cfg(cmgd_db_hndl_t db_hndl,
+        char *xpath, struct lyd_node *node, struct nb_node *nb_node,
+		void *ctxt)
+{
+	cmgd_bcknd_client_subscr_info_t subscr_info;
+	cmgd_bcknd_client_adapter_t *adptr;
+	struct nb_config_cbs *root;
+	uint32_t seq = 0;
+
+	if (cmgd_bcknd_get_subscr_info_for_xpath(xpath, &subscr_info) != 0) {
+		CMGD_BCKND_ADPTR_ERR("ERROR: Failed to get subscriber for '%s'",
+			xpath);
+		return;
+	}
+
+	adptr = (cmgd_bcknd_client_adapter_t *)ctxt;
+
+	if (!subscr_info.xpath_subscr[adptr->id].subscribed)
+		return;
+
+	root = &adptr->changes;
+	nb_config_diff_created(node, &seq, root);
```

Comment:
You are sending the same 'seq' every time. I thought it should be incrementing with every call.